src/instructlab/model/evaluate.py

```diff
@@ -498,6 +499,13 @@ def launch_server(
     is_flag=True,
     help="Print serving engine logs.",
 )
+@click.option(
+    "--device",
+    type=click.Choice(["cpu", "cuda", "mps"]),
+    default="cpu",
+    cls=clickext.ConfigOption,
+    help="PyTorch device (e.g. `cpu`, `cuda`, or `mps`) for running models.",
```

Comment:
You don't need this if you use the `clickext.ConfigOption` class, the help is inherited from the Field description.